src/graphql/queries.js

```diff
@@ -66,13 +66,16 @@ export const GET_COLLECTION_BY_ENTRY_ID = gql`
 
 export const GET_ALL_COLLECTIONS = gql`
   query GetAllCollections {
-    collectionCollection {
+    collectionGalleryCollection {
       items {
-        ...CollectionFields
+        itemsCollection {
+          items {
+            name
```

Comment:
I noticed a few differnt spots using this query, I would think the other fields are still necessary, no?
You can still use the fragment there
```
items {
...CollectionFields
}
```

Reply:
The only place this queries seems to be called is in the paintings pages. I did that search in vscode that you showed me. The query is shown in 3 other files one in paintings, one is the query call itself and one is in the contentful lib. I opted to not use the fragment because it was returning too much data that I didn't need. Also the paintings page when  loaded says: Warning: data for page "/paintings" is 208 kB which exceeds the threshold of 128 kB, this amount of data can reduce performance. So I opted to have a query that returns less data to reduce the data for the paintings page.